src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Converters/IsNullOrEmptyConverter_Tests.cs

```diff
@@ -14,7 +14,7 @@ public void IsNullOrEmptyConverter(object value, bool expectedResult)
 		{
 			var isNullOrEmptyConverter = new IsNullOrEmptyConverter();
 
-			var result = isNullOrEmptyConverter.Convert(value, typeof(IsNotNullOrEmptyConverter_Tests), null, CultureInfo.CurrentCulture);
+			var result = isNullOrEmptyConverter.Convert(value, typeof(IsNotNullOrEmptyConverter_Tests), null!, CultureInfo.CurrentCulture);
```

Comment:
Let's remove the null-forgiving operator
```suggestion
			var result = isNullOrEmptyConverter.Convert(value, typeof(IsNotNullOrEmptyConverter_Tests), null, CultureInfo.CurrentCulture);
```